### DISCUSSION TITLE
Turning off inlinegit blame by default and only enable in the file when toggled back on.

### DISCUSSION THREAD

**Discussion by BKG01** [zed-industries/zed#14330] [Feature Requests]
The inline git blame is a useful feature for when it's needed, but for me it's a distraction most of the time. 

It would be ideal to be able to turn it off by default and only enable it when select in the top right menu just whilst that buffer is open. 
This is pretty much the inverse of what seems to be the default currently, where it's on by default by can be toggled off for that buffer.   Is there a way to change this behaviour?

**Comment by SomeoneToIgnore** [comment]
See https://github.com/zed-industries/zed/issues/14354

**Reply by BKG01** [reply]
Thanks.
